package.json

```diff
@@ -51,11 +51,11 @@
     "lint": "eslint",
     "test": "scripts/test.sh",
     "package": "npm ci --build-from-source=realm && node-pre-gyp package",
-    "install": "node-pre-gyp install --fallback-to-build",
-    "rebuild": "npm run isWin -s && SET REALMJS_USE_DEBUG_CORE=1 & npm run rebuild-changes || REALMJS_USE_DEBUG_CORE=1 && npm run rebuild-changes",
-    "build": "npm run isWin -s && SET REALMJS_USE_DEBUG_CORE=1 & npm run build-changes || REALMJS_USE_DEBUG_CORE=1 && npm run build-changes",
-    "build-changes": "node-pre-gyp build --debug",
-    "rebuild-changes": "node-pre-gyp rebuild --debug && cd tests && npm install",
+    "install": "node-pre-gyp install --fallback-to-build -j 8",
```

Comment:
Does this means running 8 processes in parallel? If `node-pre-gyp` respects `MAKEFLAGS` or similar, I think it is better to leave it to the user to decide.

Reply:
The user just uses `npm install`. We should be using good default for this. -j 8 is good default imo.